### DISCUSSION TITLE
packing executables into wheels

### DISCUSSION THREAD

**Discussion by fehrin1** [mesonbuild/meson-python#712] [Q&A]
Hi all, this is my first question here, i would like to include a binary executable in a wheel in a way that on install it is transferred to the Scipts directory (i use Windows lingo/logic here), which is in the PATH (of my virtual environment).
What i mean is a general executable, no console script or entry point.
Am i on the right trail on putting my executable my.exe into pack-0.0.1-py3-none-win_amd64.whl
as (in)
pack-0.0.1.data/scripts/my.exe
?
And if so, how do i achieve this (in a sense this is the main question)?
For the complete picture i want to compile a single Fortran source to an executable and with this executable proceed as above.
Thx in advance for any hints.

**Comment by dnicolodi** [comment]
Hello @fehrin1. You don't need to do anything special, `meson-python` takes care of everything for you: just compile the binary with the `executable()` meson function and mark it as installed by default. See the example here https://github.com/mesonbuild/meson-python/tree/main/tests/packages/executable https://github.com/mesonbuild/meson-python/blob/91d64d1d08aa2702ceaf1bde3edc53d69429100e/tests/packages/executable/meson.build#L7-L11

**Comment by fehrin1** [comment]
Thx a lot, i will give it a try.
